staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go

```diff
@@ -31,11 +31,17 @@ import (
 
 const (
 	LeaderElectionRecordAnnotationKey = "control-plane.alpha.kubernetes.io/leader"
-	EndpointsResourceLock             = "endpoints"
-	ConfigMapsResourceLock            = "configmaps"
 	LeasesResourceLock                = "leases"
-	EndpointsLeasesResourceLock       = "endpointsleases"
-	ConfigMapsLeasesResourceLock      = "configmapsleases"
+	// When using EndpointsLeasesResourceLock, you need to ensure that
```

Comment:
@liggitt - I'm sure this is not the documentation you wanted to see.
Can you point me to the one you would like to be updated?

Reply:
this wouldn't be enough for me to know what I was supposed to do... do we have an example of the flow schema they'd need to set up?

Reply:
Added an example below.